examples/upscale_relperm.cpp

```diff
@@ -260,6 +267,230 @@ static void writeEclipseOutput(Lazy& RelPermValues,
+            {"surfaceTension",               "11"}, // Surface tension given in dynes/cm
+            {"output",                         ""}, // If this is set, output goes to screen and to this file.
+            {"gravity",                     "0.0"}, // default is no gravitational effects
+            {"waterDensity",                "1.0"}, // default density of water, only applicable to gravity
+            {"oilDensity",                  "0.6"}, // ditto
+            {"interpolate",                   "0"}, // default is not to interpolate
+            {"maxpoints",                  "1000"}, // maximal number of saturation points.
+            {"outputprecision",               "4"}, // number of significant numbers to print
+            {"maxPermContrast",             "1e7"}, // maximum allowed contrast in each single-phase computation
+            {"minPerm",                   "1e-12"}, // absolute minimum for allowed cell permeability
+            {"maxPerm",                  "100000"}, // maximal allowed cell permeability
+            {"minPoro",                  "0.0001"}, // this limit is necessary for pcmin/max computation
+            {"saturationThreshold",     "0.00001"}, // accuracy threshold for saturation, we ignore Pc values that
+                                                    // give so small contributions near endpoints.
+            {"linsolver_tolerance",       "1e-12"}, // residual tolerance for linear solver
+            {"linsolver_verbosity",           "0"}, // verbosity level for linear solver
+            {"linsolver_max_iterations",      "0"}, // Maximum number of iterations allow, specify 0 for default
+            {"linsolver_type",                "3"}, // Type of linear solver: 0 = ILU0/CG, 1 = AMG/CG, 2 KAMG/CG, 3 FAST_AMG/CG
+            {"linsolver_prolongate_factor", "1.0"}, // Prolongation factor in AMG
+            {"linsolver_smooth_steps",        "1"}, // Number of smoothing steps in AMG
+            {"fluids",                       "ow"}, // Whether upscaling for oil/water (ow) or gas/oil (go)
+            {"krowxswirr",                   "-1"}, // Relative permeability in x direction of oil in corresponding oil/water system
+            {"krowyswirr",                   "-1"}, // Relative permeability in y direction of oil in corresponding oil/water system
+            {"krowzswirr",                   "-1"}, // Relative permeability in z direction of oil in corresponding oil/water system
+            {"doEclipseCheck",             "true"}, // Check if minimum relpermvalues in input are zero (specify critical saturations)
+            {"critRelpermThresh",          "1e-6"}, // Threshold for setting minimum relperm to 0 (thus specify critical saturations)
+        };
+    }
+
+    template <class String>
+    std::shared_ptr<const Opm::Deck>
+    parseEclipseFile(const String& ECLIPSEFILENAME, const bool isMaster)
```

Comment:
this believe this could be (a static) in RelPermUtils. that would enable sharing with upscale_perm which was the reason i introduced the helper class.